packages/mui-base/src/AutocompleteUnstyled/useAutocomplete.js

```diff
@@ -850,6 +850,16 @@ export default function useAutocomplete(props) {
             });
           }
           break;
+        case 'Delete':
+          if (multiple && !readOnly && inputValue === '' && value.length > 0 && focusedTag !== -1) {
+            const index = focusedTag === -1 ? value.length - 1 : focusedTag;
```

Comment:
No need to check if `focusedTag === -1`, as the outer `if` handles this already.

Reply:
Awesome, thank you! I appreciate it. I went in and made the change. 😄